### PR TITLE
[AWS] Remove the need for access and secret keys in the configuration

### PR DESCRIPTION
**Pull request by JosepSampe**
Developer's Certificate of Origin 1.1

       By making a contribution to this project, I certify that:

       (a) The contribution was created in whole or in part by me and I
           have the right to submit it under the Apache License 2.0; or

       (b) The contribution is based upon previous work that, to the best
           of my knowledge, is covered under an appropriate open source
           license and I have the right under that license to submit that
           work with modifications, whether created in whole or in part
           by me, under the same open source license (unless I am
           permitted to submit under a different license), as indicated
           in the file; or

       (c) The contribution was provided directly to me by some other
           person who certified (a), (b) or (c) and I have not modified
           it.

       (d) I understand and agree that this project and the contribution
           are public and that a record of the contribution (including all
           personal information I submit with it, including my sign-off) is
           maintained indefinitely and may be redistributed consistent with
           this project or the open source license(s) involved.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,17 +3,17 @@
 ## [v3.2.1.dev0]
 
 ### Added
-- [Core] Added a mechanism to automatically retry failed tasks by @tomwhite
```
Comment (tomwhite) [inline CHANGELOG.md:6]: Any reason you removed my name?

Comment (JosepSampe) [reply CHANGELOG.md:6]: No, I was planing to put names on the release, just to avoid putting my name everywhere :)

Comment (tomwhite) [reply CHANGELOG.md:6]: I don't mind either way. Maybe do it in a separate PR which explains the motivation?

Comment (tomwhite) [reply CHANGELOG.md:6]: BTW you can get GH to generate release notes which automatically adds contributors' names. https://docs.github.com/en/repositories/releasing-projects-on-github/automatically-generated-release-notes#about-automatically-generated-release-notes

**Pull request merged**